duckduckgo/iOS
# Daniel/subscriptions/8.itp

Task/Issue URL: https://app.asana.com/0/72649045549333/1205043809052693/f
Tech Design URL:
CC:

**Description**:
- Implements ITP for subscriptions
- Implements navigating back in WebViews
- Shows/Hide navigation bar on WebViews depending on scroll position
- Subscription WebViews are now shown in a sheet for better navigation/usability
- Includes several improvements to the subscription flow for memory management and etc.
- Allows navigating to tel: FaceTime: and etc schemas
- Properly manages to navigate to new windows



**Steps to test this PR**:
⚠️  This requires a sandbox account and the alpha version of the app

1. Get a Subscription
2. Go to ITP
3. Navigate between links and views
4. Check that PDF files can be seen and shared (ℹ️ Downloaded PDF is a dummy file for testing, as real PDF's are behind internal user wall)





---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
